yt_dlp/extractor/viu.py

```diff
@@ -227,21 +210,22 @@ class ViuOTTIE(InfoExtractor):
         'zh-cn': 2,
         'en-us': 3,
     }
+
     _user_info = None
 
     def _detect_error(self, response):
         code = response.get('status', {}).get('code')
         if code > 0:
             message = try_get(response, lambda x: x['status']['message'])
-            raise ExtractorError('%s said: %s (%s)' % (
-                self.IE_NAME, message, code), expected=True)
+            raise ExtractorError(
+                f'{self.IE_NAME} said: {message} ({code})', expected=True)
         return response['data']
 
     def _raise_login_required(self):
         raise ExtractorError(
             'This video requires login. '
-            'Specify --username and --password or --netrc (machine: %s) '
-            'to provide account credentials.' % self._NETRC_MACHINE,
+            f'Specify --username and --password or --netrc (machine: {self._NETRC_MACHINE}) '
+            'to provide account credentials.',
             expected=True)
```

Comment:
this function is actually redundant

Reply:
Yeah. I only changed string format there.
But it's not required actually. It's just for old method which require token.